core/connector/src/main/java/org/exoplatform/ecm/connector/dlp/FileDlpConnector.java

```diff
@@ -227,5 +169,21 @@ public void removePositiveItem(String itemReference) {
       }
     } catch (Exception e) {
       LOGGER.error("Error while deleting dlp file item", e);
+    }
+  }
+
+  @Override
+  public String getItemUrl(String itemReference) {
+    ExtendedSession session = null;
+    try {
+      session =
+          (ExtendedSession) WCMCoreUtils.getSystemSessionProvider()
+                                        .getSession(COLLABORATION_WS, repositoryService.getCurrentRepository());
+      Node node = session.getNodeByIdentifier(itemReference);
+      return WCMCoreUtils.getLinkInDocumentsApplication(node.getPath());
```

Comment:
can show me the path generated by this  ?
Which drive is used ?

Reply:
This is an example of a path generated: /portal/dw/documents?path=Security%2FSecurity%2Flegume%252520205008770.xlsx

Reply:
Ok for me